pkg/cloudprovider/aws/block_store.go

```diff
@@ -282,7 +283,13 @@ func (b *blockStore) SetVolumeID(unstructuredPV runtime.Unstructured, volumeID s
 		return nil, errors.New("spec.awsElasticBlockStore not found")
 	}
 
-	pv.Spec.AWSElasticBlockStore.VolumeID = volumeID
+	pvFailureDomainZone := pv.Labels["failure-domain.beta.kubernetes.io/zone"]
```

Comment:
I'm wondering if `failure-domain.beta.kubernetes.io/zone` should be a constant.

Reply:
@skriss I guess this is a question for you ^

Reply:
I figured since it wasn't used more than once it wasn't buying much benefit having it as a separate variable. Finding it shouldn't too difficult, because changing it would likely mean you're directly concerned with this method. Though this string is used in other files, so maybe if this was a project or module wide it could be beneficial, but I didn't want to fit that all into a PR especially because I've not worked extensively with go.